apps/passwd.c

```diff
@@ -184,9 +184,10 @@ int passwd_main(int argc, char **argv)
             break;
         }
     }
+
+    /* All optional arguments, are the password text. */
```

Comment:
usage for this should probably have .... i.e [password....]

Reply:
Hmmm...  perhaps "All remaining arguments are the password text" ?

Reply:
@slontis, I don't understand your comment.
@levitte, thanks used your text in the comment :)

Reply:
type in
````
openssl password
````
and look at what it prints out
The last arg should be [password....] not [password]

Reply:
I got prompted for a password.  But okay, I put "..." in the options list, and tweaked the manpage.